frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -766,6 +835,107 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
                 });
             });
 
+            $('#learnRuleModal').on('show.bs.modal', function (event) {
+                submit_learn_rule($(this), initial_data['key'], initial_data['prefix'], initial_data['type'], initial_data['hijack_as'], 'show');
```

Comment:
you should do this only if active = True, otherwise go directly to ModalNoChange